pages/security/output_encoding.md

```diff
@@ -0,0 +1,43 @@
++---
```

Comment:
I think there was a copy-pasta error here. Looks like all the lines have a `+` in front of them:

```
+---
+title: Output Enco
+permalink: /security/output-encoding
+layout: docs
+sidenav: security
+---
```

Also, should the title be `Output Encoding` instead of `Output Enco`?